content/desktop/install/mac-install.md

```diff
@@ -47,11 +47,9 @@ This page contains download URLs, information about system requirements, and ins
 {{< /tab >}}
 {{< tab name="Mac with Apple silicon" >}}
 
-Beginning with Docker Desktop 4.3.0, we have removed the hard requirement to install Rosetta 2. There are a few optional command line tools that still require Rosetta 2 when using Darwin/AMD64. See [Known issues](../troubleshoot/known-issues.md). However, to get the best experience, we recommend that you install Rosetta 2. To install Rosetta 2 manually from the command line, run the following command:
+Beginning with Docker Desktop 4.17, users will be prompted to install Rosetta when you opt-in to use it but have not already installed it.
 
-  ```console
-  $ softwareupdate --install-rosetta
-  ```
+Rosetta is Generally Available now on MacOS 13 and later versions. It provides faster emulation of Intel-based images on Apple Silicon. To use Rosetta, see Settings. Beginning with Docker Desktop 4.25, Rosetta is enabled by default on macOS 14.1 and later versions.
```

Comment:
Technically this is not a system requirement, rather just an information about Rosetta. So feel free to suggest if this needs to be moved to some other section of docs.